sdk/identity/Azure.Identity.BrokeredAuthentication/src/SharedTokenCacheCredentialBrokerOptions.cs

```diff
@@ -28,11 +30,26 @@ public SharedTokenCacheCredentialBrokerOptions(TokenCachePersistenceOptions toke
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="SharedTokenCacheCredentialBrokerOptions"/>.
+        /// </summary>
+        /// <param name="tokenCacheOptions">The <see cref="TokenCachePersistenceOptions"/> that will apply to the token cache used by this credential.</param>
+        /// <param name="enableMsaPassthrough">Enables Microsoft Account (MSA) passthough.</param>
+        public SharedTokenCacheCredentialBrokerOptions(TokenCachePersistenceOptions tokenCacheOptions = null, bool enableMsaPassthrough = false)
```

Comment:
Same comment as above regarding making this a public settable property.

Reply:
Done.